app/containers/message/CallButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import Touchable from 'react-native-platform-touchable';
+import PropTypes from 'prop-types';
+
+import { formatLastMessage, BUTTON_HIT_SLOP } from './utils';
+import styles from './styles';
+import I18n from '../../i18n';
+import { CustomIcon } from '../../lib/Icons';
+import callJitsi from '../../lib/methods/callJitsi';
+import { showErrorAlert } from '../../utils/info';
+
+const call = (rid, jitsiTimeout) => {
```

Comment:
IMO this method should be on RoomView.
You won't need to mock Jitsi then and it'll behave like other methods.